BringgSDK.js

```diff
@@ -1268,6 +1278,7 @@ var BringgSDK = (function () {
       if (lastEta <= 10) { // 10 minutes away can start getting updated ETA without traffic
         log('calculating ETA less than a 10 minutes, not taking from server anymore');
         etaFromServer = false;
+        return calculateETA(originLat, originLng, destLat, destLng, destAddress, onETACalculatedCallback);
```

Comment:
can you explain this change? you are calling calculateEta with `etaFromServer = false` which will call distance matrix

Reply:
Correct. With this change, as soon as the eta reaches 10, it will start getting distance from distance matrix, which I think is the preferred behavior. Otherwise, it won't use the distance matrix until the next call to calculateETA after it goes below 10.